google/cloud/forseti/notifier/notifiers/cscc_notifier.py

```diff
@@ -223,14 +223,32 @@ def _send_findings_to_cscc(self, violations, source_id=None):
             formated_findings_in_page = (
```

Comment:
It seems a small check here would work?
```
for page in paged_findings_in_cscc:
  if not page:
    continue
  <rest of your code>
```

Reply:
`page` isn't empty even when findings are not retrieved. So this wouldn't work.